security/index.adoc

```diff
@@ -0,0 +1,85 @@
+[[security-intro]]
+= Introduction
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-intro-about-this-guide]]
+== About This Guide
+
+This guide provides a high-level walkthrough of the container security measures
+available in {product-title}, from the host layer to the container and
+orchestration layer to the user build and application layer. After reading this
+guide, you should be able to identify:
+
+- Why container security is important and how it compares with existing security standards.
+- Which container security measures are provided by the host (RHEL) layer and which are provided uniquely from {product-title}.
+- How to evaluate your container content and sources for vulnerabilities.
+- How to design your build and deployment process to proactively check container content.
+- How to control access to containers via authentication and authorization.
+- How networking and attached storage are secured in {product-title}.
+- Containerized solutions for API management and SSO.
+
+Basic concepts covered in this guide include links to further reading provided
+elsewhere in the {product-title} product documentation as well as other Red Hat
+and external documentation.
+
+[[security-intro-what-are-containers]]
+== What Are Containers?
```

Comment:
I like this, and I think it's a much better way to explain containers than what we have anywhere else in the docs. But is it the right place for it? Wouldn't it be easier to put this in the Architecture guide and just reference it here?

Note: I see you link to the Architecture guide in the"Further Reading" links down the bottom, so the reader is being directed there anyway.